app/sidekiq/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -88,20 +114,58 @@ def perform(submission_id)
 
       private
 
+      def file_body
+        @file_body ||= File.read('lib/evss/disability_compensation_form/bdd_instructions.pdf')
+      end
+
       def upload_bdd_instructions
-        client.upload(file_body, document_data)
+        # NOTE: the ApiProviderFactory implements its own Flipper,
+        # ApiProviderFactory::FEATURE_TOGGLE_UPLOAD_BDD_INSTRUCTIONS, which will be used to iteratively
+        # start sending more BDD Instructions to Lighthouse instead of EVSS
+        # However, since using this factory is new behavior, it is hidden behind this additional Flipper, which will act
+        # as a "kill switch" in the event there are problems with the ApiProviderFactory implementation, so we can
+        # revert back to using the EVSS::DocumentsService directly here
+        if Flipper.enabled?(:disability_compensation_use_api_provider_for_bdd_instructions)
+          api_response = upload_via_api_provider
+
+          if upload_provider.is_a?(LighthouseSupplementalDocumentUploadProvider)
+            create_lighthouse_polling_record(api_response)
+          end
+
+          upload_provider.log_upload_success(STATSD_KEY_PREFIX)
+        else
+          EVSS::DocumentsService.new(submission.auth_headers).upload(file_body, document_data)
+        end
       end
 
-      def file_body
-        @file_body ||= File.read('lib/evss/disability_compensation_form/bdd_instructions.pdf')
+      def upload_via_api_provider
+        document = upload_provider.generate_upload_document(
+          BDD_INSTRUCTIONS_FILE_NAME,
+          BDD_INSTRUCTIONS_DOCUMENT_TYPE
+        )
+        upload_provider.submit_upload_document(document, file_body)
       end
 
-      def client
-        @client ||= if Flipper.enabled?(:disability_compensation_lighthouse_document_service_provider)
-                      # TODO: create client from lighthouse document service
-                    else
-                      EVSS::DocumentsService.new(submission.auth_headers)
-                    end
+      def upload_provider
+        @upload_provider ||= EVSS::DisabilityCompensationForm::UploadBddInstructions.api_upload_provider(submission)
+      end
+
+      # Creates a Lighthouse526DocumentUpload record, where we save
+      # a unique 'request ID' Lighthouse provides us in the API response after we upload a document.
+      # We use this ID in the Form526DocumentUploadPollingJob chron job to check the status of the document
+      # after Lighthouse has received it.
+      #
+      # @param api_response [Faraday::Response] the response from the Lighthouse Benefits Documents API upload endpoint
+      def create_lighthouse_polling_record(api_response)
+        response_body = api_response.body['data']
+
+        if response_body['success'] == true && response_body['requestId']
```

Comment:
I was thinking initially, like is here, we shouldn't create a polling record, if the doc is not uploaded. But then I was thinking, we might still want record of that we could not upload in the DB. Like using it like a status table. But we are calling it polling, not statuses, and we do already track that somewhat in the form526_job_statuses table, so I think this is probably fine how it is.